javascript/src/main/components/TutorAssignment/TutorAssignmentForm.js

```diff
@@ -55,11 +55,16 @@ const TutorAssignmentForm = ({ createTutorAssignment, updateTutorAssignment, exi
 
     
     if(existingTutorAssignment && tutorAssignment.index === null){
-        courseList.map((course, index) => {
-            if(course.id === existingTutorAssignment.course.id){
-                setTutorAssignment({...tutorAssignment, index: index});
+        for (var i = 0; i < courseList.length; i++) {
+            if (courseList[i].id === existingTutorAssignment.course.id) {
+                setTutorAssignment({...tutorAssignment, index: i});
             }
-        });
+        }
+        // courseList.map((course, index) => {
+        //     if(course.id === existingTutorAssignment.course.id){
+                
+        //     }
+        // });
```

Comment:
I'm unsure of how this got here... do we need this?

Reply:
o shoot this is a oopsie, I was trying to see why .map triggers eslint I think